.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+jdk:
+  - oraclejdk11
+
+script:
+  - ./gradlew clean build check
+
+branches:
+  only:
+    - master
```

Comment:
Rozumiem, że to oznacza tylko przy pushach (merge'ach) na mastera oraz przy otwarciu PR? Nie chcemy na innych branchach? Chcemy? Nie wiem?

Reply:
Testy po raz pierwszy są uruchamiane przy PR, potem już każdy push na brancha związanego z PR je uruchamia. Myślę, że jak póki co nie mamy testów, a tylko sprawdzamy jakość kodu to chyba nie ma sensu je puszczać na każdym branchu po pushu. Czy może ktoś chce?

Reply:
Znaczy... mamy testy

Reply:
Faktycznie, backend ma testy. To żeby podjąć decyzję, ustawiam, żeby dla każdego brancha, którego nazwa zaczyna się od IO2019TEAM- odpalał się build. Jeżeli kogoś interesuje rezultat to rzuci na niego okiem